pages/index.tsx

```diff
@@ -140,28 +135,48 @@ export default function IndexPage({ pageProps, setReleaseNote }) {
         <Container>
           <Row>
             <Col>
-              <br />
               <h2>Reverse ETL makes your data warehouse actionable</h2>
-              <p>
-                Grouparoo connects your customer data to all the places where it
-                can make a difference. Use your data warehouse to power
-                marketing campaigns, customer service, and everything else your
-                business needs to succeed.
-                <br />
-                <small>
-                  <Link href="/solutions/reverse-etl">
-                    <a>Learn more about reverse ETL</a>
-                  </Link>
-                </small>
+
+              <br />
+              <p style={{ textAlign: "left" }}>
+                We are at the next step in the data-driven evolution of how
```

Comment:
I prefer the existing copy for this part of the page. This new description is quite wordy. I'd rather we add this copy further down on the page or elsewhere.